ERROR - 2025-09-04 02:49:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'unwi3891_kspk'@'localhost' (using password: YES) in C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'unwi3891_kspk', Object(SensitiveParameterValue), 'unwi3891_kspk', 3306, '', 0)
#1 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(3, 0)
#6 C:\xampp82\htdocs\kspk8\app\Controllers\Home.php(27): CodeIgniter\BaseModel->findAll(3)
#7 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Home->index()
#8 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp82\htdocs\kspk8\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(3, 0)
#5 C:\xampp82\htdocs\kspk8\app\Controllers\Home.php(27): CodeIgniter\BaseModel->findAll(3)
#6 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\kspk8\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-04 02:49:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `berita`
ORDER BY `berita_tanggal` DESC
 LIMIT 3', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(3, 0)
 5 APPPATH\Controllers\Home.php(27): CodeIgniter\BaseModel->findAll(3)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-09-04 02:49:28 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'unwi3891_kspk'@'localhost' (using password: YES) in C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'unwi3891_kspk', Object(SensitiveParameterValue), 'unwi3891_kspk', 3306, '', 0)
#1 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(3, 0)
#6 C:\xampp82\htdocs\kspk8\app\Controllers\Home.php(27): CodeIgniter\BaseModel->findAll(3)
#7 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Home->index()
#8 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#11 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp82\htdocs\kspk8\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(419): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\BaseModel.php(677): CodeIgniter\Model->doFindAll(3, 0)
#5 C:\xampp82\htdocs\kspk8\app\Controllers\Home.php(27): CodeIgniter\BaseModel->findAll(3)
#6 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(933): App\Controllers\Home->index()
#7 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#8 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(363): CodeIgniter\CodeIgniter->run()
#10 C:\xampp82\htdocs\kspk8\vendor\codeigniter4\framework\system\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp82\htdocs\kspk8\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-09-04 02:49:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 465.
 1 SYSTEMPATH\Database\BaseConnection.php(614): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `berita`
ORDER BY `berita_tanggal` DESC
 LIMIT 3', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(677): CodeIgniter\Model->doFindAll(3, 0)
 5 APPPATH\Controllers\Home.php(27): CodeIgniter\BaseModel->findAll(3)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-09-04 02:55:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-09-04 02:55:08 --> Error: Class "Dompdf\Options" not found
[Method: POST, Route: cetakkartu]
in APPPATH\Controllers\Home.php on line 245.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->cetakkartu()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(363): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(68): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
